axelrod/strategies/grudger.py

```diff
@@ -342,3 +342,75 @@ def strategy(opponent: Player) -> Action:
         elif opponent.defections:
             return D
         return C
+
+
+class CAPRI(Player):
```

Comment:
```suggestion
class Capri(Player):
```

This follows PEP8: the `name` of the strategy can still be `CAPRI` which is in line with other similar strategies.